test3/task2/priorityQueue.c

```diff
@@ -0,0 +1,95 @@
+#include "priorityQueue.h"
+#include <stdlib.h>
+
+typedef struct {
```

Comment:
```suggestion
typedef struct QueueNode {
```
Иначе компилятор не понимает, что за `struct QueueNode* next;` и выдаёт предупреждение в месте использования.